astrokat/observe_main.py

```diff
@@ -376,24 +379,20 @@ def run_observation(opts, kat):
             if opts.all_up and (len(catalogue.filter(el_limit_deg=opts.horizon)) != len(catalogue)):
                 raise NotAllTargetsUpError('Not all targets are currently visible - '
                                            'please re-run the script with --visibility for information')
-        cal_tags = {'Polarisation': 'polcal',
-                    'Flux': 'fluxcal',
-                    'Bandpass': 'bpcal',
-                    'Gain': 'gaincal',
-                    'Delay': 'delaycal',
-                    }
-        target_tags = []
-        for cal_type in cal_tags.keys():
-            target_tags.append('~{}'.format(cal_tags[cal_type]))
-            cal_array = [repr(cal.name) for cal in catalogue.filter(cal_tags[cal_type])]
+        # List sources and their associated functions from observation tags
+        not_cals_filter_list = []
+        for cal_type in cal_tags:
+            not_cals_filter_list.append('~{}'.format(cal_type))
+            cal_array = [repr(cal.name) for cal in catalogue.filter(cal_type)]
             if len(cal_array) < 1:
                 continue  # do not display empty tags
             cal_list = ', '.join(cal_array)
             user_logger.info("{} calibrators are [{}]".format(
-                             cal_type,
+                             str.upper(cal_type.replace('cal', '')),
```

Comment:
Hopefully no one wants to use a "calcal" type

Reply:
Thru, perhaps I wanted to be to verbose and nice
```
2019-08-01 06:55:49Z - BP calibrators are ['J0408-6545']
2019-08-01 06:55:49Z - FLUX calibrators are ['J0408-6545']
2019-08-01 06:55:49Z - GAIN calibrators are ['J0632+1022', 'J1150-0023']
2019-08-01 06:55:49Z - DELAY calibrators are ['J0408-6545']
2019-08-01 06:55:49Z - Observation targets are ['3C 175', '3C 274']
```

What if I make it sort, with zero assumptions, only outputting the tags and information as listed?
```
2019-08-01 06:55:49Z - BPCAL ['J0408-6545']
2019-08-01 06:55:49Z - FLUXCAL ['J0408-6545']
2019-08-01 06:55:49Z - GAINCAL ['J0632+1022', 'J1150-0023']
2019-08-01 06:55:49Z - DELAYCAL ['J0408-6545']
2019-08-01 06:55:49Z - TARGETS ['3C 175', '3C 274']
```

Reply:
The verbose printout is nicer.

Reply:
And sticking with verbose means @xinyuwu doesn't need to fix the tests again.

Reply:
There is also the elusive `'target'` tag that may or may not become mandatory in the near future...

Reply:
Executive decision:
If we stick with the **assumption** that calibrators will be tagged using the current format
`tag=<caltype>+"cal"`
Then the potential accidental "cal" substring from a new `caltype` name (somewhere in the future) will be inconsequential and the format definition can be applied throughout, such as
`user_logger.info("{} calibrators are {}".format(str.upper(cal_type[:-3]), cal_array))`
Providing the verbose output as previous, but not having issues with "calcal" (if the user wants to), or any naming that may contain the "cal" substring.